tox.ini

```diff
@@ -2,26 +2,26 @@
 envlist=py35
         py36
 
-[testenv:py34]
-deps =
-    nose
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/requirements_test.txt
-commands=python -m setup test
-passenv=PYPEMAN_TEST_SMTP_USER PYPEMAN_TEST_SMTP_PASSWORD PYPEMAN_TEST_RECIPIENT_EMAIL
-
 [testenv:py35]
 deps =
     nose
+    coverage
```

Comment:
why add coverage and codecov here. shouldn't 
`r{toxinidir}/requirements_test.txt`
fetch it already?

Reply:
Don't remember why but seems working without that...